test/e2e/specs/default-config/recordedit/multi-form-input.spec.js

```diff
@@ -0,0 +1,732 @@
+        column_name: 'text_col'
+      },
+      {
+        type: "input_text",
+        appy_to_all_value: "12.4",
+        formIndexes: [2,3,4,5],
+        apply_to_some:  "4.5",
+        column_name: 'float_col'
+      },
+      {
+        type: "input_text",
+        appy_to_all_value: "456",
+        apply_to_some: "123",
+        formIndexes: [2,3,4,5],
+        column_name: 'int_col'
+      },
+      {
+        type: "input_text",
+        appy_to_all_value: '2020-12-11',
+        apply_to_some: "2020-11-18",
+        formIndexes: [2,3,4,5],
+        column_name: 'date_col'
+      },
+      {
+        type: "timestamp_col",
+        appy_to_all_value_time: '12:38:31',
+        formIndexes: [2,3,4,5],
+        apply_to_some_time: '11:28:41',
+       
+        appy_to_all_value_date: '2020-12-11',
+        apply_to_some_date: '2020-11-18',
+        column_name: 'timestamp_col'
+      },
+      {
+        type: "boolean_col",
+        appy_to_all_value: 'false',
+        formIndexes: [2,3,4,5],
+        apply_to_some: 'true',
+        column_name: 'boolean_col',
+        booleanOptions: ["true", "false"]
+      },
+      {
+        type: "fk_col",
+        appy_to_all_value: 'three',
+        formIndexes: [2,3,4,5],
+        apply_to_some: 'two',
+        column_name: 'fk_col'
+      },
+      
+    ],
+    submission: {
+      table_displayname: 'main',
+      result_columns: [
+        "id", "markdown_col", "text_col", "int_col", "float_col", "date_col", "timestamp_col", "boolean_col",
+        'find the hash by looking at the th element class it will be in c_<name> format',
+        "asset_col", "asset_col_filename"
+      ],
+      results: [
+        ['1', 'some value', '', '', ''],
+        [],
+        [],
+        [],
+        []
+      ]
+    }
+  }
+}
+describe('Regarding multi form input and clone button', () => {
+  let cloneFormInput, cloneFormSubmitButton, checkboxLabel, checkboxInput;;
+  describe('Regarding multi form input,', () => {
+    describe('in create mode', () => {
+      const EC = protractor.ExpectedConditions;
+      beforeAll((done) => {
+        chaisePage.navigate(`${browser.params.url}/recordedit/#${browser.params.catalogId}/${testParams.schema_table}`);
+        chaisePage.recordeditPageReady().then(() => {
+          cloneFormInput = chaisePage.recordEditPage.getCloneFormInput();
+          cloneFormSubmitButton = chaisePage.recordEditPage.getCloneFormInputSubmitButton();
+          done();
+        }).catch(chaisePage.catchTestError(done));
+      });
+
+      it('it should not be offered in single mode.', () => {
+        let toggleBtn = recordEditPage.getColumnMultiFormButton('markdown_col');
+        expect(toggleBtn.isPresent()).toBeFalsy("toggle btn is present");
+      });
+
+      it('it should be displayed as soon as users added a new form.', (done) => {
+        chaisePage.clickButton(cloneFormSubmitButton).then(() => {
+          return recordEditPage.getColumnMultiFormButton('markdown_col')
```

Comment:
you don't need to create a promise here. just capture it in a variable and do the `isPresent`.. similar to the previous it.